react-app/src/component/ColorSelector/ColorSelectorComponent.tsx

```diff
@@ -46,7 +73,7 @@ export const ColorSelectorComponent: React.FC<legendProps> = ({
     extent = d3.extent(data, (d: any) => d?.value);
   }
 
-  const handleChange = React.useCallback(() => {
+  const handleChange = () => {
```

Comment:
Why this change? Having it as a React hook can improve rendering performance.

Reply:
It was throwing lint error(react-hooks/exhaustive-deps) so had made change but will revert it back to react hook and find some other way to resolve error.